todo.md

```diff
@@ -5,9 +5,11 @@ with definitions, translations, or whatever else you can come up with.
 ### Todo
 
 - [ ] Improve UI visuals
+- [ ] Add dark themes version of logo
 
 ### Backlog
 
+- [ ] Some regexs will be invalid (content_annotation_handler.js->createRegexMap will fail silently and annotation will not happen)

```

Comment:
Which regexs are will be invalid? Can you provide more details?

Reply:
I tried to use "+" as a key.
`
// content_annotation_handler.js at line 373
new RegExp("+", 'gi') // throws an error
`
throws an error, but it's not caught anywhere and the annotation just stops without any indication of what happened, even if there are other keys that would be fine.

Reply:
We talked about this. This has an example for how to deal with it https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_expressions#escaping